### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by lrmi0808**
Init Project initialization.

### CODE REVIEW

**Comment by lrmi0808** [comment]
Ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.4', '>= 6.1.4.1'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```
Comment (huongnt-2545) [inline Gemfile:9]: using `gem 'mysql2' instead of`

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MalayGroup1</title>
```
Comment (huongnt-2545) [inline app/views/layouts/application.html.erb:4]: I18n

File: config/database.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```
Comment (huongnt-2545) [inline config/database.yml:1]: Please follow comments by trainer at the tutorial for this
- gitignore this file
- create new file is named `database.yml.example` in `config/` folder
- config DB use MySQL for project (refs: https://edgeguides.rubyonrails.org/configuring.html#configuring-a-mysql-or-mariadb-database)

**Comment by lrmi0808** [comment]
Ready

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+# Ensure the MySQL gem is defined in your Gemfile
+#   gem 'mysql2'
+#
+# And be sure to use new-style password hashing:
+#   https://dev.mysql.com/doc/refman/5.7/en/password-hashing.html
+#
+default: &default
+  adapter: mysql2
+  encoding: utf8mb4
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
```
Comment (ngocvt-0484) [inline config/database.yml:15]: dont push this file
please add it to gitignore

**Comment by lrmi0808** [comment]
Ready

**Review by anlvq-1058 — COMMENTED** [review]
you should have file database.yml.example have content same of database.yml it may help your teammate

**Comment by lrmi0808** [comment]
Added database example.

**Comment by lrmi0808** [comment]
Ready

**Comment by lrmi0808** [comment]
> you should have file database.yml.example have content same of database.yml it may help your teammate

Added database.yml.example.

File: config/credentials.yml.enc [hunk 5]
```diff
@@ -0,0 +1 @@
+NLlaUhJagboqkR+4EwZCGZfciJaTlB9xJ65mDGN4e0Rxp4pIM6KZTL6sQZB4mXbzD+45b2uIGpcpR25Vtwxt7jotjqZun0CvD1yBKo9yvSxr+V0iD5KKXRgTvZsqA/ZWoo+aZP4IpoBT/luj7szaGwkv9+pLDGUO3JEgrOIwTGQyAroMU2AslDfZS8s/fkiOOqZFJ/naHNUHEMW5yoPupulBPlVMQFe4nZxfgU/6gs8m2Hy3WnAKLCBkZW34sl4W5gBoDFPlf2h6Rwdpq9Cm9frXw1A8IZHk/3aRpqn4/Cmtg3EWdz+WDzx21laDzo0TQjgZqzH6HmsSW3k3u7vQ8Bkwi4GFqco+ED3UtE3AqbMpm/Nl8f1pGzsEr+zXdl2pb987R0TgGYWQItzwBB4pfWeZgSapBD+B6BiN--vQ35SBZO+2APok+5--qF8np37Os/FxLN0ZfcJcSA==
```
Comment (anlvq-1058) [inline config/credentials.yml.enc:1]: please use git ignore to ignore this file

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by lrmi0808** [comment]
Ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**